django/db/models/expressions.py

```diff
@@ -477,6 +477,35 @@ def desc(self, **kwargs):
         return OrderBy(self, descending=True, **kwargs)
 
 
+class ResolvedOuterRef(F):
+    """
+    An object that contains a reference to an outer query.
+
+    In this case, the reference to the outer query has been resolved, because the
+    inner query has been used as a subquery.
+    """
+    def as_sql(self, *args, **kwargs):
+        raise ValueError(
```

Comment:
I have actually done this: but because it referred to the first line in the statement (which did not actually get changed), then GitHub thinks I haven't changed it.